failure_derive won't build with quote 1.0.3

My CI builds just started failing with the following error:

```
error[E0433]: failed to resolve: could not find `__rt` in `quote`
   --> /Users/sloveless/.cargo/registry/src/github.com-1ecc6299db9ec823/failure_derive-0.1.6/src/lib.rs:107:70
    |
107 | fn display_body(s: &synstructure::Structure) -> Result<Option<quote::__rt::TokenStream>, Error> {
    |                                                                      ^^^^ could not find `__rt` in `quote`
```

Looks like the `quote` crate renamed `__rt` to `__private` in its `1.0.3` release that just came out: https://github.com/dtolnay/quote/commit/41543890aa76f4f8046fffac536b9445275aab26

--- a/failure_derive/src/lib.rs
+++ b/failure_derive/src/lib.rs
@@ -104,7 +104,7 @@ fn fail_derive_impl(s: synstructure::Structure) -> Result<TokenStream, Error> {
     })
 }
 
-fn display_body(s: &synstructure::Structure) -> Result<Option<quote::__rt::TokenStream>, Error> {
+fn display_body(s: &synstructure::Structure) -> Result<Option<TokenStream>, Error> {
     let mut msgs = s.variants().iter().map(|v| find_error_msg(&v.ast().attrs));
     if msgs.all(|msg| msg.map(|m| m.is_none()).unwrap_or(true)) {
         return Ok(None);

